src/main/groovy/org/edx/jenkins/dsl/JenkinsPublicConstants.groovy

```diff
@@ -221,19 +209,4 @@ class JenkinsPublicConstants {
             buildButton()
         }
     }
-
-    // Reusable closure for configuring a masked password user parameter with a default value
-    // Input must be structured in the following format:
-    // [ name: String x, description: String y, default: String z ]
-    public static final Closure JENKINS_PUBLIC_MASKED_PASSWORD = { param ->
-        return {
-            it / 'properties' / 'hudson.model.ParametersDefinitionProperty' / parameterDefinitions << 'hudson.model.PasswordParameterDefinition' {
-                name param.name
-                defaultValue Secret.fromString(param.default.toString()).getEncryptedValue()
-                description param.description
-            }
-        }
-
-    }
-
```

Comment:
Are you sure we should pull this out? I *think* it was used by loadtest or gdpr

Reply:
I don't see it in use anywhere..